src/engine/client/favorites.cpp

```diff
@@ -226,7 +226,7 @@ const CFavorites::CEntry *CFavorites::Entry(const NETADDR &Addr) const
 void CFavorites::RemoveEntry(CEntry *pEntry)
 {
 	// Replace the entry
-	int Index = pEntry - &m_aEntries[0];
+	int Index = pEntry - (m_aEntries).data();
```

Comment:
```suggestion
	int Index = pEntry - m_aEntries.data();
```

Clang-tidy fix is not very smart about this.

Reply:
Looked sus to me too. But I did not feel comfy doubting mr clang.